src/screens/Messages/Tabs/TeamChat.js

```diff
@@ -74,16 +74,16 @@ class TeamChat extends Component {
 
     removeTeam = async (room) => {
 
-        let token = await AsyncStorage.getItem("token")
-        let user = await AsyncStorage.getItem("userID")
+        let userID = "eaf54fae-1ab8-4b5a-8047-51904f6ae884"
```

Comment:
do these need to be changed for the reports to be generated? It shouldn't be changed for the actual app, but only for testing